docs/src/troubleshooting.md

```diff
@@ -42,15 +30,60 @@ Adjusting `PATH` can differ based on the operating system and the default shell
 </tr>
 </table>
 
-The exact changes to make can differ depending on what program you want to expose to Sublime Text. The simplest way is to extend the path like so (replacing `/usr/local/bin` with the path of your choice):
+> **Note**: It might be necessary to re-login your user account after changing the shell initialization script for the changes to be picked up.
+
+
+Another solution could be (at least on Linux) to update the server `PATH` using the `env`parameter in your **LSP** configuration file. The following template can be used where:
+  - `<your_language_server_name>` is the server name
+  - `<added_path>` is the directory needed for the server to behave correctly
+
+```json
+"<your_language_server_name>":
+{
+    // ...
+
+    "env":
+    {
+        "PATH": "<added_path>:/usr/local/bin"
+    }
+}
+```
+
+## Common problems
 
+### 1. LSP doesn't start my language server
+
+* Make sure you have a folder added in your Sublime workspace.
+* Make sure the document you are opening lives under that folder.
+
+Your client configuration requires two settings to match the document your are editing:
+
+* Scope (eg. `source.php`): Verify this is correct by running "Show Scope Name" from the developer menu.
+* Syntax (eg. `Packages\PHP\PHP.sublime-syntax`): Verify by running `view.settings().get("syntax")` in the console.
+
+### 2. LSP cannot find my language server (`No such file or directory: 'xyz'`)
+Assuming that the server is actually installed, and that you can start it from your shell, this issue is likely due to Sublime Text's internal environment not picking up the same `PATH` environment variable as you've configured in your shell.
+
+The exact changes to make can differ depending on what program you want to expose to Sublime Text. The simplest way is to extend the path like so (replacing `/usr/local/bin` with the path of your choice):
```

Comment:
The `The exact changes to make` paragraph (including the code block) should be in the `Updating the PATH used by LSP servers` section. Just maybe add info that this is a Linux specific method.